cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -84,7 +86,13 @@ instance HasSeverityAnnotation NtN.AcceptConnectionsPolicyTrace where
 
 instance HasPrivacyAnnotation (TraceFetchClientState header)
 instance HasSeverityAnnotation (TraceFetchClientState header) where
-  getSeverityAnnotation _ = Info
+  getSeverityAnnotation BlockFetch.AddedFetchRequest {} = Info
+  getSeverityAnnotation BlockFetch.AcknowledgedFetchRequest {} = Info
+  getSeverityAnnotation BlockFetch.StartedFetchBatch {} = Info
+  getSeverityAnnotation BlockFetch.CompletedBlockFetch {} = Info
+  getSeverityAnnotation BlockFetch.CompletedFetchBatch {} = Info
+  getSeverityAnnotation BlockFetch.RejectedFetchBatch {} = Info
+  getSeverityAnnotation BlockFetch.ClientTerminating {} = Notice
 
```

Comment:
@coot, do you think it can be:
```suggestion
  getSeverityAnnotation BlockFetch.ClientTerminating {} = Notice
  getSeverityAnnotation _ = Info
```
?

Reply:
We will have something like 20 hot peers, so these messages will rather be sparse. I thought that initially its better to start with `Notice` and we can move to `Info` later. Are you  ok with it @deepfire ?